backend/src/main/java/com/github/JeTSkY1h/book/BookController.java

```diff
@@ -21,6 +22,15 @@
 public class BookController {
     private final BookService bookService;
 
+    @PutMapping("/api/book/rate/{rating}")
```

Comment:
URLs should identify resources. The rating is not the resource, the book is. I would use `/api/books/{id}` as the URL and would use it for all changes that happen to books.